app/controllers/api/v0/classified_listings_controller.rb

```diff
@@ -30,10 +31,18 @@ def index
       end
 
       def show
-        @classified_listing = ClassifiedListing.
-          select(ATTRIBUTES_FOR_SERIALIZATION).
-          includes(:user, :organization).
-          find(params[:id])
+        relation = ClassifiedListing.published
+
+        # if the user is authenticated we allow them to access
+        # their own unpublished listings as well
+        relation = if @user
+                     relation.union(@user.classified_listings)
+                   else
+                     relation.includes(:user, :organization)
```

Comment:
Maybe I'm mistaken, wouldn't you want to include these regardless of whether or not user is authenticated?

Reply:
Great catch! Neither matters I as we use `.find()` to load a single object. Going to test it now

Reply:
Removed :)